docs/conf.py

```diff
@@ -119,3 +119,162 @@
 # Tell sphinx-autodoc-typehints to generate stub parameter annotations including
 # types, even if the parameters aren't explicitly documented.
 always_document_param_types = True
+
+
+# -- Patch sphinx.ext.autosummary.generate ----------------------------------
```

Comment:
Did you write all this code yourself, or did you just modify a specific line? I think it is useful to add a link to the original code and mark which lines you changed in this patch.

Reply:
As stated a little bit below in that comment, this patch only adds a single line to `generate_autosummary_content`:

```python
ns['annotations'] = list(getattr(obj, '__annotations__', {}).keys())
```
I've added the link to the source to the comment.

Reply:
Ahh apologies! I wasn't really clear to me what that line meant, thanks for the clarification.